MarkdownConverter/src/System/Tasks/ConversionRunner.ts

```diff
@@ -1,6 +1,7 @@
 import Clone = require("clone");
 import { CultureInfo } from "culture-info";
 import Template = require("es6-template-string");
+import Fs = require("fs");
```

Comment:
I know it's peanuts but I'd rather like this variable to be named `FileSystem` rather than `Fs`  to make it matching better to the general variable-naming.